test/data/search_ad_link_data.html

```diff
@@ -0,0 +1,20 @@
+<html>
+<head>
+  <title>Brave Search</title>
+  <script type="application/ld+json">
+    {
+      "@context": {
+         "@vocab": "http://schema.org/",
+         "uuid": null,
+         "creativeInstanceId": null,
+         "creativeSetId": null
+      },
+      "@type": "Product",
```

Comment:
what does this type refer to?

Reply:
chromium supports very few of json-ld types: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/document_metadata/document_metadata_extractor.cc;l=61?q=metadata_extractor.cc&ss=chromium

I have found that `Product` is quite suitable for our purposes. `Product` type is also defined on schema.org: https://schema.org/Product